### ISSUE TITLE
Plaintext governance proposal type (with independent tally parameters)

### ISSUE THREAD

**Issue by gavinly** [anoma/namada#3477]
we think that **plaintext (or "signalling") proposals** should **replace the default proposal type** so that the Namada community can use token voting for social signalling, independent from the current default proposal type that currently supports wasm execution. we think that proposals capable of wasm execution should have a high bar to clear, whereas plaintext proposals should not. perhaps we could **give wasm-capable proposals a name that better reflects their power**

as mentioned, **independent tallying parameters** would be another important aspect to this feature, similar to: https://github.com/anoma/namada/issues/3475

**Comment by gavinly** [comment]
could we pls add to https://github.com/anoma/namada/milestone/26

**Comment by brentstone** [comment]
Do you envision just renaming these things to be sufficient? Default proposals without wasm code are effectively already the kinds of proposals you describe.

**Comment by gavinly** [comment]
yes i think that should work, as long as default proposals without wasm are a lower quorum threshold and require a simple majority to pass

having it named distinctly should be helpful for designing interfaces and alerts. some people may want special attn to PGF, whereas others may want them filtered out
